server/server.go

```diff
@@ -477,14 +497,20 @@ func (s *Server) setClusterName(name string) {
 		resetCh <- struct{}{}
 	}
 	s.Noticef("Cluster name updated to %s", name)
-
 }
 
 // Return whether the cluster name is dynamic.
 func (s *Server) isClusterNameDynamic() bool {
 	return s.getOpts().Cluster.Name == ""
 }
 
+// Return whether the cluster name is susceptible to cluster name changes.
+func (s *Server) isClusterNameSusceptibleToChanges() bool {
```

Comment:
again, in this case could be `isClusterNameChangeAllowed()`

Reply:
This name change should probably be applied to `s.susceptible` as well.